pkg/route/registry/route/rest.go

```diff
@@ -103,6 +110,13 @@ func (rs *REST) Update(ctx kapi.Context, obj runtime.Object) (<-chan apiserver.R
 	if errs := validation.ValidateRoute(route); len(errs) > 0 {
 		return nil, errors.NewInvalid("route", route.Name, errs)
 	}
+
+	err := rs.loadCerts(route.TLS)
```

Comment:
Ok, cool.  The only consideration I ran in to while testing is that it requires the line breaks.  Will update and test.